t/inline-init4.neon

```diff
@@ -0,0 +1 @@
+VAR a: Number := 5
```

Comment:
Add a comment here saying that there is no output tested because you're specifically testing a bug that happened when the variable was *not* referenced later.